src/main/java/org/aludratest/config/impl/SimplePreferences.java

```diff
@@ -29,11 +29,16 @@
  */
 public class SimplePreferences extends AbstractPreferences implements MutablePreferences {
 
-    private Map<String, String> values = new HashMap<String, String>();
+    private Map<String, String> values;
```

Comment:
No, have to correct myself. According to the description, I bet this relates to children, not to values.